package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@duckduckgo/autoconsent": "^4.1.1",
-    "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#6.4.3",
+    "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#shane/pwg-01",
```

Comment:
Will need a stable `duckduckgo-autofill` release before we're finished instead of targeting this WIP branch